CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Grafana Mimir
 
+* [CHANGE] Alertmanager: CLI flag `-utf8-strict-mode` has been added, and its default value is set to `true`. #6898
```

Comment:
```suggestion
* [FEATURE] Alertmanager: Added `-alertmanager.utf8-strict-mode` to control support for any UTF-8 character as part of Alertmanager configuration/API matchers and labels. It's default value is set to `false`. #6898
```

Reply:
I think this is a feature more than a change, we tend to use whole flag and the default value is actually false.

Reply:
Good catch on the true/false. I copied another CHANGELOG as an example and forgot to change this.